src/i18n/zh-TW.json

```diff
@@ -15,5 +15,38 @@
     "remember": "記住我",
     "forget": "忘記密碼?",
     "login": "登入"
+  },
+  "settingpage": {
+    "pageTitle": "設定",
+    "oldPassword": "輸入舊密碼",
+    "newPassword": "輸入新密碼",
+    "confirmPassword": "再次輸入新密碼",
+    "changePassword": "更改密碼",
+    "changelanguage": "ENGLISH",
```

Comment:
Don't set English words in Chinese setting file.

Reply:
done